choreography/src/delivery/Dockerfile

```diff
@@ -45,14 +45,10 @@ RUN useradd -m -s /bin/bash -U $user
 
 WORKDIR /app
 COPY --from=publish /app/out ./
-COPY scripts/. ./
-RUN \
-    # Ensures the entry point is executable
-    chmod ugo+x /app/run.sh
 
 RUN chown -R $user.$user /app
 
 # Set it for subsequent commands
 USER $user
 
-ENTRYPOINT ["/bin/bash", "/app/run.sh"]
```

Comment:
I'd rather delete the script files if we plan not to use them from here.

Reply:
Sounds good. I'll delete them.